client/dashboard/profile-wizard/steps/plugins.js

```diff
@@ -2,50 +2,215 @@
 
 /**
  * Internal depdencies
  */
-import { H, Stepper } from '@woocommerce/components';
+import { H, Stepper, Card } from '@woocommerce/components';
 import ProfileWizardHeader from '../header';
 
-export default class Start extends Component {
+const plugins = [ 'jetpack', 'woocommerce-services' ];
+
+class Plugins extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.state = {
+			step: 'install',
+			isPending: true,
+			isError: false,
+			pluginsInstalled: 0,
+			pluginsActivated: 0,
+			connectUrl: '',
+		};
+
+		this.activatePlugins = this.activatePlugins.bind( this );
+	}
+
+	componentDidMount() {
+		this.installPlugins();
+	}
+
+	componentDidUpdate( prevProps, prevState ) {
+		if (
+			this.state.pluginsInstalled !== prevState.pluginsInstalled &&
+			this.state.pluginsInstalled === plugins.length
+		) {
+			/* eslint-disable react/no-did-update-set-state */
+			this.setState( {
+				step: 'activate',
+				isPending: false,
+			} );
+			/* eslint-enable react/no-did-update-set-state */
+		}
+
+		if (
+			this.state.pluginsActivated !== prevState.pluginsActivated &&
+			this.state.pluginsActivated === plugins.length
+		) {
+			this.connectJetpack();
+		}
+	}
+
+	installPlugins() {
+		forEach( plugins, async plugin => {
+			const response = await this.doPluginAction( 'install', plugin );
+			if ( 'success' === response.status ) {
+				this.setState( state => ( {
+					pluginsInstalled: state.pluginsInstalled + 1,
+				} ) );
+			}
+		} );
+	}
+
+	activatePlugins( event ) {
+		event.preventDefault();
+
+		// Avoid double activating.
+		const { isPending } = this.state;
+		if ( isPending ) {
+			return false;
+		}
+
+		this.setState( {
+			isPending: true,
+		} );
+
+		forEach( plugins, async plugin => {
+			const response = await this.doPluginAction( 'activate', plugin );
+			if ( 'success' === response.status ) {
+				this.setState( state => ( {
+					pluginsActivated: state.pluginsActivated + 1,
+				} ) );
+			}
+		} );
+	}
+
+	getErrorMessage( action, plugin ) {
+		return 'install' === action
+			? sprintf(
+					__( 'There was an error installing %s. Please try again.', 'woocommerce-admin' ),
+					this.getPluginName( plugin )
+				)
+			: sprintf(
+					__( 'There was an error activating %s. Please try again.', 'woocommerce-admin' ),
+					this.getPluginName( plugin )
+				);
+	}
+
+	async doPluginAction( action, plugin ) {
+		try {
+			const pluginResponse = await apiFetch( {
+				path: `/wc-admin/v1/onboarding/plugins/${ action }`,
+				method: 'POST',
+				data: {
+					plugin,
+				},
+			} );
+			return pluginResponse;
+		} catch ( err ) {
+			this.props.addNotice( {
+				status: 'error',
+				message: this.getErrorMessage( action, plugin ),
+			} );
+			this.setState( {
+				isPending: false,
+				isError: true,
+			} );
+		}
+	}
+
+	async connectJetpack() {
+		try {
+			const connectResponse = await apiFetch( {
+				path: '/wc-admin/v1/onboarding/plugins/connect-jetpack',
```

Comment:
Minor, and i'm guessing we won't rev these endpoints much ( if ever since they are internal only ) but do we want to have a shared namespace as a constant across all the components akin to what we have done elsewhere in wc-admin?